scripts/entrypoint-wrapper.sh

```diff
@@ -8,8 +8,23 @@ mkdir -p /var/tmp/runner/_work
 RUNNER_DATE=$(date +"%y-%m-%d")
 head /dev/urandom | tr -dc a-f0-9 | head -c 13 | sed 's/$/-'$RUNNER_DATE'/' > /var/tmp/runner/_work/runner_id.txt
 _RUNNER_ID="$(cat /var/tmp/runner/_work/runner_id.txt)"
+# The name is a bit long, but there's no other way to give the runner a unique name with a scaled docker-compose runner
 RUNNER_NAME="${RUNNER_NAME}-${_RUNNER_ID}"
 RUNNER_WORKDIR="${RUNNER_WORKDIR}/${_RUNNER_ID}"
 
+# Get hardware info and append to labels
```

Comment:
This creates some overlap with the downstream hardware env vars in Lurk, which ideally could be removed and accessed directly from `{{ env.LABELS }}` somehow

Reply:
I think labels are fundamentally selectors for CI jobs.

While I like that this is much more x-platform than in prior iterations, those are strings that are not necessarily great as labels. 
- can `uname -ms` return something with a comma?
- can I really select against 121.57 GB RAM?
- is it wise to select against 32 vCPUs without also selecting on the CPU class?

IOW to me the labels would contain stuff that's much simpler (e.g. gpu label if there's one GPU, linux label if uname matches one of a few strings) than what we probe and display in CI.